src/main/java/io/cdap/delta/bigquery/MultiGCSWriter.java

```diff
@@ -76,7 +77,8 @@ public synchronized void write(Sequenced<DMLEvent> sequencedEvent) {
     DMLEvent event = sequencedEvent.getEvent();
     DMLOperation dmlOperation = event.getOperation();
     Key key = new Key(event.getOperation().getDatabaseName(), dmlOperation.getTableName());
-    TableObject tableObject = objects.computeIfAbsent(key, t -> new TableObject(event.getOperation().getDatabaseName(),
+    TableObject tableObject = objects.computeIfAbsent(key, t -> new TableObject(dmlOperation.getDatabaseName(),
```

Comment:
TableObject is representing something written to GCS and then loaded to BigQuery. While big query doesn't have a schema concept.

I think it would be btter to add schema to the Key instead of the Table Object.